client/components/deposits-overview/next-deposit.tsx

```diff
@@ -40,23 +40,6 @@ const NextDepositDetails: React.FC< NextDepositProps > = ( {
 
 	return (
 		<>
-			{ /* Next Deposit Heading */ }
-			<div className="wcpay-deposits-overview__heading">
-				<span className="wcpay-deposits-overview__heading__title">
-					<Loadable
-						isLoading={ isLoading }
-						value={ strings.nextDeposit.title }
-					/>
-				</span>
-
-				<span className="wcpay-deposits-overview__heading__description">
-					<Loadable
-						isLoading={ isLoading }
-						value={ strings.nextDeposit.description }
-					/>
-				</span>
-			</div>
```

Comment:
I moved this section of the **`<NextDepositDetails>`** component so it could be reused fro the **Deposit History** title and description.